app/assets/javascripts/discourse/app/components/modal/dismiss-read.gjs

```diff
@@ -11,18 +11,21 @@ export default class DismissRead extends Component {
 
   <template>
     <DModal
+      @bodyClass={{unless @model.showStopTracking "empty"}}
```

Comment:
Need this to work around the fact that the `:empty` CSS selector doesn't work because EmberJS injects empty comments and spaces where we have the `if` conditional.